tools/ci/travis-install.py

```diff
@@ -0,0 +1,73 @@
+import os
+import shutil
+import subprocess
+
+def which(program):
+    def is_exe(fpath):
+        return os.path.isfile(fpath) and os.access(fpath, os.X_OK)
+
+    fpath, fname = os.path.split(program)
+    if fpath:
+        if is_exe(program):
+            return program
+    else:
+        for path in os.environ["PATH"].split(os.pathsep):
+            path = path.strip('"')
+            exe_file = os.path.join(path, program)
+            if is_exe(exe_file):
+                return exe_file
+    return None
+
+
+def install(omp=False, mpi=False, phdf5=False, dagmc=False):
+    # Create build directory and change to it
+    shutil.rmtree('build', ignore_errors=True)
+    os.mkdir('build')
+    os.chdir('build')
+
+    # Build in debug mode by default
+    cmake_cmd = ['cmake', '-Ddebug=on']
+
+    # Turn off OpenMP if specified
+    if not omp:
+        cmake_cmd.append('-Dopenmp=off')
+
+    # Use MPI wrappers when building in parallel
+    if mpi:
+        os.environ['CXX'] = 'mpicxx'
+
+    # Tell CMake to prefer parallel HDF5 if specified
+    if phdf5:
+        if not mpi:
+            raise ValueError('Parallel HDF5 must be used in '
+                             'conjunction with MPI.')
+        cmake_cmd.append('-DHDF5_PREFER_PARALLEL=ON')
+    else:
+        cmake_cmd.append('-DHDF5_PREFER_PARALLEL=OFF')
+
+    if dagmc:
+        cmake_cmd.append('-Ddagmc=ON')
+
+    # Build in coverage mode for coverage testing
+    cmake_cmd.append('-Dcoverage=on')
+
+    # Build and install
+    cmake_cmd.append('..')
+    print(' '.join(cmake_cmd))
```

Comment:
This looks like a debugging print that should be removed.

Reply:
This print is left here intentionally so that the cmake command shows in the test log.